### ISSUE TITLE
Don't remove link with attribute data-embed

### ISSUE THREAD

**Issue by tomkrej** [jonkemp/gulp-inline-css#66]
Hi,
I have small wish :) I am newsletter developer a I use gulp-inline-css in my devstack with two cases. First I have external link to style.css (<link rel="stylesheet" href="../css/style.css">) which I need to inline and remove. And second external link for Google Fonts (<link href="https://fonts.googleapis.com/css?family=Roboto:300,500,700&display=swap&subset=latin-ext" rel="stylesheet" data-embed>) which I don't want remove. 

I tried to add data attribute data-embed, but it still remove element.

Can you help me, please?

**Comment by howeller** [comment]
I second this request. My links to to Google fonts get stripped out.

**Comment by jonkemp** [comment]
This option doesn't work?

https://github.com/jonkemp/gulp-inline-css#optionsremovelinktags

**Comment by jonkemp** [comment]
Oh I see what you are saying. Seems like `data-embed` is the solution here but maybe it doesn't work for the link tag.

**Comment by howeller** [comment]
Yes - that would be a big help for HTML email dev. We use a variety of CSS between inlining, linking to external style sheets, and embedding css inside a style tag in the head. Thank you for looking into this!

**Comment by xxxKOTxxx** [comment]
Any updates?
Same problem with Google fonts

**Comment by Chris-Reedy** [comment]
Curious if this is being addressed? Also having a problem with Google Fonts being stripped.

**Comment by cb911nr** [comment]
Also came here for the same issue, want to ignore certain link tags, like google fonts.

**Comment by Chris-Reedy** [comment]
### **UPDATE**
If you use `data-embed=""` in a Google webfont `<link>`, it won't be stripped.  Here are the steps...

**Step 1**: make sure `rel="stylesheet"` isn't in the `<link>` attribute (_will interrupt gulp-inline-css_)
**Step 2**: add `inline-webfontlink=""` as a place holder so we can add the `rel="stylesheet"` back in later or else the webfont won't render (_see 1st image, below_)
**Step 3**: add `data-embed=""` to the `<link>` after the `inline-webfontlink=""` declaration (_see 1st image, below_)
**Step 4**: make sure your gulp-inline-css settings are correct (_see 2nd image, below_)
**Step 5**: add `.pipe(replace(/inline-webfontlink="(.*)"/g, 'rel="stylesheet"'))` as the last item in your `var inline = function(path) {` (_see 3rd image, below_)
**Step 6**: view source of rendered code to confirm (_see 4th image, below_)

<ins>1. HTML</ins>: 

![image](https://github.com/jonkemp/gulp-inline-css/assets/9635730/94837e48-06d7-4a7f-8dfc-80dbc620b1ff)

<ins>2. Settings</ins>:

![image](https://github.com/jonkemp/gulp-inline-css/assets/9635730/ae9795d3-11b7-4c19-9154-78d949cf2210)

<ins>3. var inline</ins>:

![image](https://github.com/jonkemp/gulp-inline-css/assets/9635730/c0dda470-f233-43be-a2f2-d9f11b3abe95)

<ins>4. Rendered</ins>:

![image](https://github.com/jonkemp/gulp-inline-css/assets/9635730/693d3564-c5ad-412c-939b-48e7e53ea66d)


_Credit in part to @chrisgrouge_
